puppetboard/app.py

```diff
@@ -176,13 +177,23 @@ def node(node_name):
     node. This includes facts and reports but not Resources as that is too
     heavy to do within a single request.
     """
+    def classes(name=None):
+        for resource in puppetdb.catalog(name).resources:
```

Comment:
What happens when `name=None`? If that's never supposed to happen, it shouldn't be a `kwarg`.